IBM/ibm-aix/ingest/parser.yml

```diff
@@ -23,7 +23,7 @@ pipeline:
           FILE_Rename: '(?<event_action>FILE_Rename)\s*%{NOTSPACE:user_name}\s*%{NOTSPACE:user_group}\s*%{NOTSPACE:process_user_name}\s*%{NOTSPACE:status}\s*%{AIX_Time}\s*No associated roles\s*frompath:\s*%{DATA:file_path} topath:%{DATA:file_name}'
           FILE_Read: '(?<event_action>FILE_Read)\s*%{NOTSPACE:process_name}\s*%{NOTSPACE:user_group}\s*%{NOTSPACE:process_user_name}\s*%{NOTSPACE:status}\s*%{AIX_Time}\s*No associated roles\s*file descriptor = 1635083369 filename =\s*%{NOTSPACE:file_name} object read event detected\s*%{NOTSPACE}'
           FILE_Pipe: '(?<event_action>FILE_Pipe)\s*%{NOTSPACE:user_name}\s*%{NOTSPACE:user_group}\s*%{NOTSPACE:process_user_name}\s*%{NOTSPACE:status}\s*%{AIX_Time}\s*No associated roles\s*read:\s*%{INT} write:\s*%{INT}'
-          USER_Login: '(flags: 80, libpath: , file:\s*%{DATA:file_name} )?(?<event_action>USER_Login)\s*%{NOTSPACE:process_name}\s*%{NOTSPACE:user_group}\s*%{NOTSPACE:process_user_name}\s*%{NOTSPACE:status}\s*%{AIX_Time}\s*No associated roles(\s*user:\s*%{NOTSPACE:user_name} tty:\s*%{NOTSPACE:process_tty})?'
+          USER_Login: '(flags: 80, libpath: , file:\s*%{DATA:file_name} )?(?<event_action>USER_Login)\s*%{NOTSPACE:process_name}\s*%{NOTSPACE:user_group}\s*%{NOTSPACE:process_user_name}\s*%{NOTSPACE:status}\s*%{AIX_Time}\s*No associated roles(\s*user:\s*%{NOTSPACE:user_name} tty:\s*#%{NOTSPACE:process_tty})?'
```

Comment:
I think it's error-prone to add the sharp in the pattern.
I prefer to not set the ECS field `process.tty.char_device.major`

Reply:
@squioc: in that last commit https://github.com/SEKOIA-IO/intake-formats/pull/499/commits/1a6b5bf6e95617321fbb3d3925e6cc530d58f2a6, I added the char `?` so that `#` becomes optional ([associated doc](https://github.com/kkos/oniguruma/blob/41a3b802af2155eef6d648aa3608e39605110642/doc/RE#L136)).

What do you think ?